### PR TITLE
Fix: Video widget creating space inside it's wrapper [ED-9809]

### PR DESCRIPTION
**Pull request by arielk**

### CODE REVIEW

**Review by StyleShit — DISMISSED** [review]

**Comment by hein-obox** [comment]
Screenshot updated, because the video widget has changed.

File: tests/playwright/sanity/video.test.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+		const wpAdmin = new WpAdminPage( page, testInfo );
+		await wpAdmin.setExperiments( {
+			container: false,
+		} );
+	} );
+
+	test( 'Verify that there is no gap between the video widget and the container', async ( { page }, testInfo ) => {
+		// Arrange.
+		const wpAdmin = new WpAdminPage( page, testInfo );
+		const editor = await wpAdmin.useElementorCleanPost(),
```
Comment (louiswol94) [inline tests/playwright/sanity/video.test.js:27]: This `const` before `editor` is not needed.

File: tests/playwright/sanity/video.test.js [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+		// Act.
+        // Set container padding to 0.
+        await editor.selectElement( containerId );
+        await editor.activatePanelTab( 'advanced' );
+        await page.locator( '.elementor-control-padding .elementor-control-dimension input' ).first().fill( '0' );
+
+        const container = await editor.getFrame().locator( `.elementor-element-${ containerId }` ),
+            containerHeight = await container.boundingBox().height,
+            videoIframeHeight = await editor.getFrame().locator( `.elementor-element-${ videoId } iframe` ).boundingBox().height;
+
```
Comment (louiswol94) [inline tests/playwright/sanity/video.test.js:40]: Looks like some indentation issues.

File: tests/playwright/sanity/video.test.js [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+		const wpAdmin = new WpAdminPage( page, testInfo );
+		await wpAdmin.setExperiments( {
+			container: false,
+		} );
+	} );
+
+	test( 'Verify that there is no gap between the video widget and the section', async ( { page }, testInfo ) => {
+		// Arrange.
+		const wpAdmin = new WpAdminPage( page, testInfo );
+		const editor = await wpAdmin.useElementorCleanPost(),
```
Comment (louiswol94) [inline tests/playwright/sanity/video.test.js:69]: Same feedback as in the container test.

**Review by louiswol94 — CHANGES_REQUESTED** [review]

File: tests/playwright/sanity/video.test.js (hunk 3 above)
Comment (hein-obox) [reply tests/playwright/sanity/video.test.js:69]: Updated.

File: tests/playwright/sanity/video.test.js (hunk 2 above)
Comment (hein-obox) [reply tests/playwright/sanity/video.test.js]: Updated

File: tests/playwright/sanity/video.test.js [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+        await editor.activatePanelTab( 'advanced' );
+        await page.locator( '.elementor-control-padding .elementor-control-dimension input' ).first().fill( '0' );
+
+        const container = editor.getFrame().locator( `.elementor-element-${ containerId }` ),
+            containerHeight = await container.boundingBox().height,
+            videoIframeHeight = await editor.getFrame().locator( `.elementor-element-${ videoId } iframe` ).boundingBox().height;
+
+		// Assert.
+		// Verify that the container has an equal height to the video iFrame.
+		expect( containerHeight ).toBe( videoIframeHeight );
```
Comment (louiswol94) [inline tests/playwright/sanity/video.test.js:45]:
```suggestion
		expect( containerHeight ).toEqual( videoIframeHeight );
```
Better readability

**Review by louiswol94 — CHANGES_REQUESTED** [review]

File: tests/playwright/sanity/video.test.js (hunk 4 above)
Comment (hein-obox) [reply tests/playwright/sanity/video.test.js]: Updated

**Review by louiswol94 — APPROVED** [review]

**Pull request merged**